static/js/tools/scatter/place_and_type_options.tsx

```diff
@@ -71,25 +77,31 @@ const CHILD_PLACE_TYPES = {
 function PlaceAndTypeOptions(): JSX.Element {
   const { place, isLoading, display } = useContext(Context);
   const [childPlaceTypes, setChildPlaceTypes] = useState([]);
-  useEffect(() => {
-    if (place.value.enclosingPlace.dcid) {
-      updateChildPlaceTypes(
-        place.value.enclosingPlace.dcid,
-        setChildPlaceTypes
-      );
-    }
-  }, [place.value.enclosingPlace.dcid]);
 
   /**
-   * Reloads child places if the enclosing place or child place type changes.
+   * Watch and update place info
    */
   useEffect(() => {
-    if (place.value.enclosingPlace.dcid && _.isNull(place.value.parentPlaces)) {
+    if (!place.value.enclosingPlace.dcid) {
+      return;
```

Comment:
does the context need to be updated in this case?

Reply:
nothing needs to happen in this case because it means no place has been chosen

Reply:
Nothing needs to happen in this case because it just means no place has been chosen yet

Reply:
could you add a comment to that effect?

Reply:
done